templates/agent/puppet.conf.erb

```diff
@@ -33,3 +33,9 @@
     noop              = <%= scope.lookupvar('::puppet::agent_noop') %>
     configtimeout     = <%= scope.lookupvar('::puppet::configtimeout') %>
     usecacheonfailure = <%= scope.lookupvar('::puppet::usecacheonfailure') %>
+<% if scope.lookupvar('::puppet::prerun_command') and !scope.lookupvar('::puppet::prerun_command').to_s.empty? -%>
+    prerun_command    = <%= scope.lookupvar('::puppet::prerun_command') %>
```

Comment:
Sometimes (maybe depending on how the class is called) undef gets passed through as nil.. or maybe that's just in rspec-puppet, so this would work.  But often with scope.lookupvar, undef will return :undef or :undefined...